priv/templates/nimble_template/Dockerfile.eex

```diff
@@ -16,7 +16,7 @@ RUN apk update && \
 <%= if web_project? do %>
 RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
 RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.34-r0/glibc-2.34-r0.apk
-RUN apk add glibc-2.34-r0.apk
+RUN apk add --force-overwrite glibc-2.34-r0.apk
```

Comment:
Hopefully the --force-overwrite will work, if yes, please also consider to upgrade to `glibc-2.35-r0.apk` too @Nihisil 👍